code/modules/projectiles/projectile/bullets/shotgun.dm

```diff
@@ -16,15 +16,30 @@
 	damage = 5
 
 /obj/item/projectile/bullet/shotgun_stunslug
-	name = "stunslug"
+	name = "electroslug"
 	damage = 5
-	stamina = 20
-	knockdown = 100
+	stamina = 30
```

Comment:
***why are we touching this one?***
Like 100% why? As it is, its a bit of a harmfull disable shot + a kd that is rather light. When used its by nukies to down the captain/hos from being able to run away

Reply:
They just bodied people and made most shit like the HoS' taser look like a joke. And being a bullet and not energy, bypass typically energy based defenses. A shotgun full of these is an easy win aa you stunlock someone to death.

Reply:
So you buffed it???

Reply:
I made them good but not overwhelmingly so. The tase effect doesn't stack and only refreshes, and since they don't use knockdown they aren't rocket tag i-win bullets. And the electrocute doesn't stun either.